modular_newhorizons/master_files/code/modules/reagents/Chemistry-Reagents/Chemistry-Recipes.dm

```diff
@@ -0,0 +1,50 @@
+/datum/chemical_reaction/mortaphenyl
+	name = "Tramadol"
+
+/datum/chemical_reaction/perconol
+	name = "Paracetamol"
+
+/datum/chemical_reaction/oxycomorphine
+	name = "Oxycodone"
+
+/datum/chemical_reaction/cetahydramine
+	name = "Diphenhydramine"
+
+/datum/chemical_reaction/asinodryl
+	name = "Ondansetron"
+
+/datum/chemical_reaction/verunol
+	name = "Ipecac"
+
+/datum/chemical_reaction/polysomnine
+	name = "Chloral Hydrate"
+
+/datum/chemical_reaction/fluvectionem
+	name = "Calomel"
+
+/datum/chemical_reaction/cataleptinol
+	name = "Mannitol"
+
+/datum/chemical_reaction/corophenidate
+	name = "Methylphenidate"
```

Comment:
Personal take:
No need to change back the psych chem names I would say. This and the ones under it.
Especially due to the fact they are more "general psych usage" now.